### PR TITLE
Add GithubService to list contributors

### PR DESCRIPTION
**Pull request by shiraji**
## Issue
#78 

## Overview (Required)
Add API network feature for getting contributors

## Screenshot
No Views have created yet.

### CODE REVIEW

**Comment by konifar** [comment]
:eyes:

File: app/src/main/java/io/github/droidkaigi/confsched2017/api/DroidKaigiClient.java [hunk 1]
```diff
@@ -47,9 +58,20 @@ private static Gson createGson() {
...
+
     interface DroidKaigiService {
 
         @GET("/2017/sessions.json")
         Single<List<Session>> getSessionsJa();
     }
+
+    interface GithubService {
+
+        @GET("/repos/{owner}/{repo}/contributors?per_page=100")
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched2017/api/DroidKaigiClient.java]: I think `per_page=100` is not necessary.

**Comment by konifar** [comment]
LGTM!
Thank you for contribution! 💯

**Comment by shiraji** [comment]
I sent the fix...I will do that later commit, then.

**Pull request merged**